source/daplink/interface/target_flash.c

```diff
@@ -45,7 +45,11 @@ static const flash_intf_t flash_intf = {
     target_flash_uninit,
     target_flash_program_page,
     target_flash_erase_sector,
+#ifdef TARGET_FLASH_ERASE_CHIP_OVERRIDE
+    board_target_flash_erase_chip,
```

Comment:
I would recommend using flash_intf_target_custom (as you mentioned earlier) instead of this.  That way this code is completely contained within the microbit board folder.